docs/source/basic_usage.rst

```diff
@@ -64,8 +64,19 @@ Ophyd objects so that we can load them in a variety of contexts. If you do not
 use ``happi`` you will need to create your objects and displays in the same
 process.
 
-Here is a quick example if you wanted to get a feel for what ``typhos`` looks
-like with `happi``:
+From the command-line, using typhos and happi together is easy.  For example,
+to load an auto-generated typhos screen for your device named ``"my_device"``
+would only require the following:
+
+.. code:: bash
+
+   $ typhos my_device
```

Comment:
I wonder how many people have typed `typhos load my_device` like me, after all the `happi load my_device`